fetch/api/response/multi-globals/incumbent/incumbent.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
Ditto for this one.
A single file defining window.createRedirectResponse and window.createRequest might reduce the code size.